antora-ui-camel/src/css/frontpage.css

```diff
@@ -237,6 +237,10 @@ section.frontpage h1 {
     flex: 100%;
   }
 
+  .box-new {
```

Comment:
Can't we use the existing `box` class? Naming things `-new` leads to naming the next new thing `-newer` and the thing after that one `-newest` and so on. This is never a good practice. Read [this](https://www.w3.org/QA/Tips/goodclassnames).

Reply:
This PR is regarding [website - Add section to display the 6 camel projects with logos](https://issues.apache.org/jira/browse/CAMEL-14772).
I created the icons and the design according to the tweet mentioned the issue.
I need help with SVG icons. Can you please help me with SVG icons?

Reply:
We have Apache Camel logo [here](https://github.com/apache/camel/tree/master/docs/img) not sure how Claus created that image in the tweet.